src/components/tag/tag.hbs

```diff
@@ -8,6 +8,3 @@
 {{#each tags}}
 <span class="{{@root.prefix}}--tag {{@root.prefix}}--tag--{{type}}">{{label}}</span>
 {{/each}}
-{{#if componentsX}}
```

Comment:
just wondering, we're no longer going to have a disabled tag example?

Reply:
No. Since its a state it won't be on preview per designer request.

Reply:
No. It won't be on preview example per designer request